website/docs/docs/build/saved-queries.md

```diff
@@ -8,16 +8,18 @@ tags: [Metrics, Semantic Layer]
 
 Saved queries are a way to save commonly used queries in MetricFlow. You can group metrics, dimensions, and filters that are logically related into a saved query. 
 
-To define a saved query, refer to the following specification:
+They are distinct from [Exports](/docs/use-dbt-semantic-layer/exports), which are scheduled and executed saved queries using [dbt's job scheduler](/docs/deploy/job-scheduler).
 
- Parameter | Description | Type |
-| --------- | ----------- | ---- |
-| `name` | The name of the metric. | Required |
-| `description` | The description of the metric. | Optional |
-| `query_params` | The query parameters for the saved query: `metrics`, `group_by`, and `where`. | Required |
+| Feature      | Saved queries  | Exports    |
+|--------------|----------------|------------|
+| **Availability**   | Available in both dbt Core and dbt Cloud.  | Available to dbt Cloud users on Team and Enterprise plans. | 
+| **Purpose**    | To define and manage common Semantic Layer queries.     | Automatically runs saved queries and materializes them within your data platform using dbt's job scheduler. |
+| **Usage**   | For organizing and reusing queries within dbt projects. | For materializing query results in the data platform.  |
 
```

Comment:
for SQ - am i right in thinking the only way to use them is via their editor or command line?

Reply:
You can also query a saved query via our APIs, and we're exposing them in downstream tools like Tableau.  Roxi has a cool demo of this!